src/applications/bmqstoragetool/m_bmqstoragetool_journalfileprocessor.t.cpp

```diff
@@ -478,9 +479,9 @@ static void test7_searchPartiallyConfirmedMessagesTest()
     }
     expectedStream << partiallyConfirmedGUIDS.size()
                    << " message GUID(s) found." << bsl::endl;
-    float messageCount     = ceil(k_NUM_RECORDS / 3.0);
-    float outstandingRatio = float(partiallyConfirmedGUIDS.size() + 1) /
-                             messageCount * 100.0;
+    const size_t messageCount     = k_NUM_RECORDS / 3 + 1;
```

Comment:
Here, if `k_NUM_RECORDS == 3`:
the previous code will have `messageCount == 1.0f`
the updated code will have `messageCount == 2`

This should help
```suggestion
    const size_t messageCount     = (k_NUM_RECORDS + 2) / 3;
```

Reply:
Nice suggestion! The test won't break if we change `k_NUM_RECORDS` in the future